app/views/ish_manager/cities/show.haml

```diff
@@ -4,23 +4,60 @@
 -#
   
 = render 'header', :city => @city
-= render 'ish_manager/features/index',  :features => @city.features,   :city => @city
-= render 'ish_manager/newsitems/index', :newsitems => @city.newsitems, :city => @city
-
-.row
-  .col-sm-6= render 'ish_manager/reports/index', :reports => @city.reports
-  .col-sm-6
-    %h5 Galleries
-    = render 'ish_manager/galleries/index', :galleries => @city.galleries
 .row
-  .col-sm-6
+  .col-sm-4
     %h5
       Venues (#{@city.venues.length})
       = link_to image_new, new_venue_path(city_id: @city.id)
+    = render 'ish_manager/venues/search', :city => @city
     = render 'ish_manager/venues/index', :venues => @city.venues
-.row
-  .col-sm-6= render 'ish_manager/videos/index', :videos => @city.videos
-  .col-sm-6
-    = render 'ish_manager/users/index', :users => @city.current_users
-    = render 'ish_manager/events/index', :events => @city.events    
-    = render 'ish_manager/venues/index', :venues => @city.venues
+  .col-sm-8
+    .col-sm-3
+      %a{href: '#newsitems'}
+        NewsItems (#{@city.newsitems.length})
+    .col-sm-3
+      %a{href: '#reports'}
+        Reports (#{@city.reports.length})
+    .col-sm-3
+      %a{href: '#galleries'}
+        Galleries (#{@city.galleries.length})
+    .col-sm-3
+      %a{href: '#videos'}
+        Videos (#{@city.videos.length})
+    .row
+      .col-sm-12#newsitems
+        = render 'ish_manager/newsitems/index', :newsitems => @city.newsitems, :city => @city
+    .row
+      .col-sm-12#reports
+        = render 'ish_manager/reports/index', :reports => @city.reports
+    .row
+      .col-sm-12#galleries
+        %h5
+          Galleries (#{@city.galleries.length})
+          = link_to image_new, new_gallery_path(city_id: @city.id)
+        = render 'ish_manager/galleries/index', :galleries => @city.galleries
```

Comment:
I think you want to make the @galleries variable in the controller, like: `@galleries = @city.galleries.page( params[:galleries_page] ).per( Gallery::PER_PAGE )